routes/nodebb.js

```diff
@@ -130,6 +136,39 @@ app.get(`${BASE_REPORT_URL}/user/username/:username`, proxyObject());
 app.post(`${BASE_REPORT_URL}/user/v1/create`, proxyObject());
 app.get(`${BASE_REPORT_URL}/user/uid/:uid`, proxyObject());
 
+function isEditablePost() {
+  logger.info({message: "isEditablePost method called"});
+  return function(req, res, next) {
+    logger.info(req.body);
+    const uid = req.body.uid;
+    const pid = req.body.pid;
+    const url = `${req.protocol}://${req.get('host')}${BASE_REPORT_URL}/post/pid/${pid}`
+    const options = {
+      url: url,
+      method: 'GET',
+      json: true
+    };
+    logger.info(options)
+      request(options, (error, response, body) => {
```

Comment:
Why are we going with `request` object here?

We can go with the proxy object itself to call the API right.

Reply:
I tried the above one by replacing request with proxy 
the proxy object itself is not calling and if I use the same proxy method for routes it is working 
but if call proxy method separately it is not working